certified-connectors/nps.today/readme.md

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
@HazemSaeid , can you please refer to this [Readme.md](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/custom-connectors/AzureKeyVault/Readme.md) example. Readme.md is more for the open-source contributors and developers for building your connector.